src/layer/index.ts

```diff
@@ -185,13 +190,35 @@ export class UserLayer extends RefCounted {
   }
 
   static supportsPickOption = false;
+  static supportsLayerBarColorSyncOption = false;
 
   pick = new TrackableBoolean(true, true);
 
   selectionState: UserLayerSelectionState;
 
   messages = new MessageList();
 
+  layerBarUserDefinedColor = new TrackableOptionalRGB();
```

Comment:
How does a user set this color for a layer? Can they do it in the JSON state? Or should we remove this?